FireRender.Maya.Src/FireRenderVolume.cpp

```diff
@@ -471,7 +468,7 @@ bool NorthstarRPRVolume::TranslateVolume()
 		volumeShader.xSetValue(RPR_MATERIAL_INPUT_COLOR, albedoLookupNode);
 	}
 
-	if (vdata.emissionGrid.IsValid()) // grid exists
+	if (vdata.emissionGrid.IsValid() && vdata.emissionGrid.gridOnIndices.size() > 0) // grid exists
```

Comment:
Please consider moving check gridOnIndices.size() > 0 to IsValid() method.

If I understood correctly grid with gridOnIndices.size() == 0 would be invalid, right?